<!DOCTYPE html>
<html>
<head>
<title>small_molecule.md</title>
<meta http-equiv="Content-type" content="text/html;charset=UTF-8">

<style>
/* https://github.com/microsoft/vscode/blob/master/extensions/markdown-language-features/media/markdown.css */
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

body {
	font-family: var(--vscode-markdown-font-family, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif);
	font-size: var(--vscode-markdown-font-size, 14px);
	padding: 0 26px;
	line-height: var(--vscode-markdown-line-height, 22px);
	word-wrap: break-word;
}

#code-csp-warning {
	position: fixed;
	top: 0;
	right: 0;
	color: white;
	margin: 16px;
	text-align: center;
	font-size: 12px;
	font-family: sans-serif;
	background-color:#444444;
	cursor: pointer;
	padding: 6px;
	box-shadow: 1px 1px 1px rgba(0,0,0,.25);
}

#code-csp-warning:hover {
	text-decoration: none;
	background-color:#007acc;
	box-shadow: 2px 2px 2px rgba(0,0,0,.25);
}

body.scrollBeyondLastLine {
	margin-bottom: calc(100vh - 22px);
}

body.showEditorSelection .code-line {
	position: relative;
}

body.showEditorSelection .code-active-line:before,
body.showEditorSelection .code-line:hover:before {
	content: "";
	display: block;
	position: absolute;
	top: 0;
	left: -12px;
	height: 100%;
}

body.showEditorSelection li.code-active-line:before,
body.showEditorSelection li.code-line:hover:before {
	left: -30px;
}

.vscode-light.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(0, 0, 0, 0.15);
}

.vscode-light.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(0, 0, 0, 0.40);
}

.vscode-light.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

.vscode-dark.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(255, 255, 255, 0.4);
}

.vscode-dark.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(255, 255, 255, 0.60);
}

.vscode-dark.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

.vscode-high-contrast.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(255, 160, 0, 0.7);
}

.vscode-high-contrast.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(255, 160, 0, 1);
}

.vscode-high-contrast.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

img {
	max-width: 100%;
	max-height: 100%;
}

a {
	text-decoration: none;
}

a:hover {
	text-decoration: underline;
}

a:focus,
input:focus,
select:focus,
textarea:focus {
	outline: 1px solid -webkit-focus-ring-color;
	outline-offset: -1px;
}

hr {
	border: 0;
	height: 2px;
	border-bottom: 2px solid;
}

h1 {
	padding-bottom: 0.3em;
	line-height: 1.2;
	border-bottom-width: 1px;
	border-bottom-style: solid;
}

h1, h2, h3 {
	font-weight: normal;
}

table {
	border-collapse: collapse;
}

table > thead > tr > th {
	text-align: left;
	border-bottom: 1px solid;
}

table > thead > tr > th,
table > thead > tr > td,
table > tbody > tr > th,
table > tbody > tr > td {
	padding: 5px 10px;
}

table > tbody > tr + tr > td {
	border-top: 1px solid;
}

blockquote {
	margin: 0 7px 0 5px;
	padding: 0 16px 0 10px;
	border-left-width: 5px;
	border-left-style: solid;
}

code {
	font-family: Menlo, Monaco, Consolas, "Droid Sans Mono", "Courier New", monospace, "Droid Sans Fallback";
	font-size: 1em;
	line-height: 1.357em;
}

body.wordWrap pre {
	white-space: pre-wrap;
}

pre:not(.hljs),
pre.hljs code > div {
	padding: 16px;
	border-radius: 3px;
	overflow: auto;
}

pre code {
	color: var(--vscode-editor-foreground);
	tab-size: 4;
}

/** Theming */

.vscode-light pre {
	background-color: rgba(220, 220, 220, 0.4);
}

.vscode-dark pre {
	background-color: rgba(10, 10, 10, 0.4);
}

.vscode-high-contrast pre {
	background-color: rgb(0, 0, 0);
}

.vscode-high-contrast h1 {
	border-color: rgb(0, 0, 0);
}

.vscode-light table > thead > tr > th {
	border-color: rgba(0, 0, 0, 0.69);
}

.vscode-dark table > thead > tr > th {
	border-color: rgba(255, 255, 255, 0.69);
}

.vscode-light h1,
.vscode-light hr,
.vscode-light table > tbody > tr + tr > td {
	border-color: rgba(0, 0, 0, 0.18);
}

.vscode-dark h1,
.vscode-dark hr,
.vscode-dark table > tbody > tr + tr > td {
	border-color: rgba(255, 255, 255, 0.18);
}

</style>

<style>
/*

Brown Paper style from goldblog.com.ua (c) Zaripov Yura <yur4ik7@ukr.net>

*/

.hljs {
  display: block;
  overflow-x: auto;
  padding: 0.5em;
  background:#b7a68e url(./brown-papersq.png);
}

.hljs-keyword,
.hljs-selector-tag,
.hljs-literal {
  color:#005599;
  font-weight:bold;
}

.hljs,
.hljs-subst {
  color: #363c69;
}

.hljs-string,
.hljs-title,
.hljs-section,
.hljs-type,
.hljs-attribute,
.hljs-symbol,
.hljs-bullet,
.hljs-built_in,
.hljs-addition,
.hljs-variable,
.hljs-template-tag,
.hljs-template-variable,
.hljs-link,
.hljs-name {
  color: #2c009f;
}

.hljs-comment,
.hljs-quote,
.hljs-meta,
.hljs-deletion {
  color: #802022;
}

.hljs-keyword,
.hljs-selector-tag,
.hljs-literal,
.hljs-doctag,
.hljs-title,
.hljs-section,
.hljs-type,
.hljs-name,
.hljs-strong {
  font-weight: bold;
}

.hljs-emphasis {
  font-style: italic;
}

</style>

<style>
/*
 * Markdown PDF CSS
 */

 body {
	font-family: -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif, "Meiryo";
	padding: 0 12px;
}

pre {
	background-color: #f8f8f8;
	border: 1px solid #cccccc;
	border-radius: 3px;
	overflow-x: auto;
	white-space: pre-wrap;
	overflow-wrap: break-word;
}

pre:not(.hljs) {
	padding: 23px;
	line-height: 19px;
}

blockquote {
	background: rgba(127, 127, 127, 0.1);
	border-color: rgba(0, 122, 204, 0.5);
}

.emoji {
	height: 1.4em;
}

code {
	font-size: 14px;
	line-height: 19px;
}

/* for inline code */
:not(pre):not(.hljs) > code {
	color: #C9AE75; /* Change the old color so it seems less like an error */
	font-size: inherit;
}

/* Page Break : use <div class="page"/> to insert page break
-------------------------------------------------------- */
.page {
	page-break-after: always;
}

</style>

<script src="https://unpkg.com/mermaid/dist/mermaid.min.js"></script>
</head>
<body>
  <script>
    mermaid.initialize({
      startOnLoad: true,
      theme: document.body.classList.contains('vscode-dark') || document.body.classList.contains('vscode-high-contrast')
          ? 'dark'
          : 'default'
    });
  </script>
<h1 id="small-rna-targeting-molecule-design">Small RNA-Targeting Molecule Design</h1>
<p><b>Identify, validate and optimize</b> small molecules that target the functional transcriptome</p>
<hr>
<h2 id="introduction">Introduction</h2>
<p>rna二级结构的重要性、历史、研究意义, diffculties in this area</p>
<p>predict structure,determine function, predict binding selectivity</p>
<p>It's a key to design an accurate model of RNA strcuture to modulate small molecules.<br>
Computational methods can model RNA structure
from sequence, including <strong>free energy minimization and phylogenetic comparison</strong>.</p>
<h3 id="methods-to-model-rna-structure">Methods to model RNA structure :</h3>
<ul>
<li>
<p>Free energy minimization: Assume fucntional structure is in minimum free energy structure. Calculating the <strong>free energy of each substructure</strong>,such as pairs,bulgers and loops.Algorithm such as <strong>mFold,RNAFold,RNAStructure</strong> incorporating these thermodynamic paramteres deteremined by experiments.(~70% accuracy)</p>
</li>
<li>
<p>Chemical modification : This will inducing pause of reverse transcriptase and mutation.</p>
</li>
<li>
<p>Biophysical 3D assay : X-ray crystallography, NMR
spectrometry and cryo-electron microscopy (cryo-EM)</p>
</li>
<li>
<p>Phylogenetic comparison(conservation) : Genetic differnce preserved in secondary structure Covariation-based  structural prediction is highly acccurate(compared to crystal structures)</p>
</li>
</ul>
<p><strong>R-scape</strong></p>
<ul>
<li>
<p>Homology modelling: 是一种用于预测蛋白质三维结构的方法，它基于“如果两个蛋白质具有足够高的序列相似性，它们很可能具有非常相似的三维结构”的原理. 同源建模通常使用已知结构的蛋白质作为模板，根据目标蛋白质的氨基酸序列，通过模拟和优化的方法构建目标蛋白质的三维结构.</p>
</li>
<li>
<p>3D structure prediction : Predicting program :  <strong>FARFAR2,MC-Fold/MC-Sym and iFoldRNA, RNAdolf,Sfold,CONTRAfold</strong> Scoring function for structure with unknown native structure(stable 3d structure): Rosetta, RASP, <strong>ARES</strong>(Outperform others, Verified by RMS to known crystal structure)</p>
</li>
</ul>
<h3 id="assessing-quality-of-predictions">Assessing quality of predictions</h3>
<p>Without a known structure, it is hard to assess the reliability and accuracy of the prediction, which has been addressed by development of various statistical methods.</p>
<ul>
<li>
<p>Partition fucntion : It contains all of the thermodynamic infomation of system. This fucntion has benn incorporated into various programs.( <strong>RNAfold, Sfold and CONTRAfold</strong>)</p>
</li>
<li>
<p>Statistics : Phylogenetic comparisons have applied statistics to demonstrate the significance in evolutionary covariation, associating with the fucntionality.</p>
</li>
</ul>
<h3 id=""></h3>
<p>上文介绍了如何分析RNA structure， 还有在功能上的分析，本部分先跳过，之后再分析</p>
<h3 id="selectivity-of-small-molecules-tagerting-dna">Selectivity of small molecules tagerting DNA</h3>
<p>It's intractable to construct small molecules that be selective for target RNA.</p>
<ul>
<li>Lack of structural diversity</li>
<li>Anionic backbone(???)</li>
<li>Diffculties in high-throughput screen</li>
<li>Huge differences between predicting strcuture/function and binding affinity</li>
</ul>
<p>Factors that affect the selectivity :</p>
<ul>
<li>Uniqueness of the structure in the transcriptome</li>
<li>Expression of the target compared with off-targets(same affinity, highly expressed target will occupy ligands)</li>
<li>Relative affinity of the small molecule for on-and-off-targets</li>
<li>Accessibility of the target site</li>
<li>Functionality of the binding site</li>
</ul>
<p>Notice that : Compounds can be designed that target multiple sites within an RNA target simultaneously, thus overcoming the limitations of structural degeneracy(a structure is not unique in the transcriptome).</p>
<h2 id="approaches-for-indentifing-small-molecule-rna-binders">Approaches for indentifing small-molecule RNA binders</h2>
<h2 id="target-centric-approaches">Target-centric approaches</h2>
<p>Target-centric means that for the small molecules' only choice for binding are single or a few targets.</p>
<ul>
<li>Affinity mass spectrometry : <strong>ALIS</strong>(automated ligand
identification system) uses indirect dectection of target-ligand interaction before liquid chromatography.</li>
</ul>
<p>It's similar to AS-MS(Affinity selection mass spectrometry, direct identification of target-ligan complexs by mass spectrometry) widely used in proteins.</p>
<ul>
<li>
<p>Fluoscence-based assays :  This is a high-throughput assay relies on displacement of a fluorescent dye or compound by a small molecule of interest.</p>
</li>
<li>
<p>Microarray-based screening :</p>
</li>
<li>
<p>Fragment-based ligand dicovery :  Fragment-based ligand discovery uses libraries of low-molecular-weight compounds to efficiently explore chemical space that might bind the target of
interests.</p>
</li>
<li>
<p>DNA-encoded compound libraries ：</p>
</li>
</ul>
<hr>
<h1 id="methods-applied-in-paper--targeting-sars-cov-2">Methods applied in paper : Targeting SARS-CoV-2</h1>
<ul>
<li>
<p>Determining binding site of SARS-CoV-2, which is an <strong>attenuator hairpin(AH)</strong></p>
<p>From previous experience, small molecule binders have prefrence for RNA hairpin strcuture over other structures including internal loops, bulges and base pairs.<br>
Also, notice that [1]<br>
[1]: ss.com</p>
</li>
<li>
<p>Verifying the structure of binding target :</p>
<ul>
<li>
<p>Free energy minimization &amp; Phylogenetic conservation<br>
Via <strong>ScanFold and RNAz</strong>, we can determine and verify 3D structure of binding site</p>
</li>
<li>
<p>Structural mapping data
Via <strong>DMS-Mapseq</strong></p>
</li>
</ul>
</li>
<li>
<p>Identify candidates of small molecule binders :</p>
<ul>
<li>Physiochemical properties of known ligands binding target<br>
(Inforna database, the RNA-binding ligand data set)</li>
</ul>
</li>
<li>
<p>Screening the binders :</p>
<ul>
<li>Microarray-based screening, <strong>AbsorbArray</strong></li>
</ul>
</li>
</ul>
<hr>
<h1 id="research-of-qiangfeng-zhang">Research of Qiangfeng Zhang</h1>
<h2 id="prismnet">PrismNet</h2>
<h2 id="smrtnet">SmrtNet</h2>

</body>
</html>
